### PR TITLE
Adding Model file for Forums database

### PR DESCRIPTION
**Pull request by DanWaldman**

### CODE REVIEW

**Comment by DanWaldman** [comment]
HELP
I do not understand why ForumModel won't import from UserModel, tried uploading them together but still didnt work, what can I do?

**Comment by Yarboa** [comment]
> HELP I do not understand why ForumModel won't import from UserModel, tried uploading them together but still didnt work, what can I do?

Did you run flake tests on your local vm?

File: AcadeMeData/ForumModels.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+from UserModels import *
```
[the remark this answers is not in the document]
Comment (Yarboa) [reply AcadeMeData/ForumModels.py:1]: please replace this line with the following line:
rom django.db import models
from django.conf import settings

to verify and search for hints 
while your vm is up, vagrant ssh
cd /vagrant
run the following:

pipenv run flake8 --max-line-length 120
and check there are no errors

**Review by Yarboa — COMMENTED** [review]
In general
* Please add migration files to migration dir
* see my review for the verification of your code 
* once it will pass CI, add tests

File: AcadeMeData/ForumModels.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+
+class MessageBoards(models.Model):
+    ID = models.OneToOneField(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, primary_key=True)
```
Comment (Omeramsc) [inline AcadeMeData/ForumModels.py:5]: Variable names / fields should be in lower case letters (apply to all)

File: AcadeMeData/ForumModels.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    CourseName = models.CharField(max_length=30)
+    Rate = models.CharField(max_length=1, choices=range(1-5), default=3)
```
Comment (Omeramsc) [inline AcadeMeData/ForumModels.py:7]: `rating` may be a better name

Comment (Omeramsc) [reply AcadeMeData/ForumModels.py:7]: and do we want to have a default rating of 3? seems like it is better be defaulted as None

**Review by Omeramsc — CHANGES_REQUESTED** [review]
Please add tests and an informative commit / PR message, including the issue it is addressing.

**Comment by Yarboa** [comment]
What is the difference between this PR and #30 they have the same name :/

**Pull request closed without merging**